sympy/combinatorics/fp_groups.py

```diff
@@ -118,7 +118,7 @@ def identity(self):
     def __contains__(self, g):
         return g in self.free_group
 
-    def subgroup(self, gens, C=None):
+    def subgroup(self, gens, C=None, homomorphism=False):
         '''
         Return the subgroup generated by `gens` using the
         Reidemeister-Schreier algorithm
```

Comment:
This should have an explanation of `homomorphism` and actually an example of `homomorphism=True` could be useful (just to demonstrate that the homomorphism returns an image of an element of `subgroup(...)` in the parent group)